lib/ansible/modules/cloud/vmware/vmware_host_facts.py

```diff
@@ -0,0 +1,178 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Wei Gao <gaowei3@qq.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_host_facts
+short_description: Gathers facts about remote vmware host
+description:
+    - Gathers facts about remote vmware host
+version_added: 2.5
+author:
+    - Wei Gao (@woshihaoren)
+notes:
+    - Tested on vSphere 5.5
```

Comment:
vSphere 5.5 is pretty old. I understand the need to disclaim any support for what is not tested, but often these statements tend to stay around for a long time despite working for other versions. So please remove it.

You end up with people believing the module is an old module and it doesn't reflect any improvements.

@Akasurde We need to remove these statements, or add them as comments note exposed in the documentation.

Reply:
ok